src/app/accessibility/accessibility-settings.service.ts

```diff
@@ -0,0 +1,316 @@
+import { Injectable } from '@angular/core';
+import cloneDeep from 'lodash/cloneDeep';
+import {
+  Observable,
+  of,
+  switchMap,
+} from 'rxjs';
+import {
+  map,
+  take,
+} from 'rxjs/operators';
+
+import { environment } from '../../environments/environment';
+import { AuthService } from '../core/auth/auth.service';
+import { EPersonDataService } from '../core/eperson/eperson-data.service';
+import { EPerson } from '../core/eperson/models/eperson.model';
+import { CookieService } from '../core/services/cookie.service';
+import { getFirstCompletedRemoteData } from '../core/shared/operators';
+import {
+  hasValue,
+  isNotEmpty,
+} from '../shared/empty.util';
+import { createSuccessfulRemoteDataObject$ } from '../shared/remote-data.utils';
+
+/**
+ * Name of the cookie used to store the settings locally
+ */
+export const ACCESSIBILITY_COOKIE = 'dsAccessibilityCookie';
```

Comment:
We need to add this Cookie to our Orejime settings (in the "Cookie Settings" in the footer).  It should be possible to opt-out of having this cookie saved, as it's not required.

NOTE: In DSpace 8 and 7, this cookie will need to be added to the Klaro settings, as we only use Orejime as of DSpace 9.  But, the two are similar.